src/KafkaW.cxx

```diff
@@ -130,6 +133,18 @@ PollStatus PollStatus::Err() {
   return ret;
 }
 
+PollStatus PollStatus::EOP() {
+  PollStatus ret;
+  ret.state = -2;
```

Comment:
PollStatus is kind of a poor man's variant type, the status code corresponds to that static function's name, in this case EOP end-of-partition.